src/resources/views/crud/inc/form_save_buttons.blade.php

```diff
@@ -13,7 +13,7 @@
             <button id="btnGroupDrop1" type="button" class="btn btn-success dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span class="caret"></span><span class="sr-only">&#x25BC;</span></button>
             <div class="dropdown-menu" aria-labelledby="btnGroupDrop1">
```

Comment:
I think this is a good opportunity to rename ```#btnGroupDrop1``` into something better. I'll do it.

Reply:
Done.